Packs/Cyberint/Integrations/Cyberint/Cyberint.py

```diff
@@ -504,20 +540,41 @@ def fetch_incidents(client: Client, last_run: Dict[str, int],
                         attachments.append(tmp_attachment)
 
         alert["attachments"] = attachments
-        alert_csv_id = alert.get('alert_data', {}).get('csv', {}).get('id', '')
-        if alert_csv_id:
-            extracted_csv_data = extract_data_from_csv_stream(client, alert_id,  # type: ignore
-                                                              alert_csv_id)
-            alert['alert_data']['csv'] = extracted_csv_data
+        alert_data = dict_safe_get(alert, ['alert_data', 'csv'])
+        incident_csv_records = dict_safe_get(alert, ['alert_data', 'csv', 'content']) or []

```

Comment:
```suggestion
        alert_data = dict_safe_get(alert, ['alert_data', 'csv'], {})
        incident_csv_records = alert_data.get('content', [])
```
We can take advantage of the 'default_return_value' argument of dict_safe_get function. 
In addition, it seems that we accessing again the keys of alert_data so why not just use this variable?